src/components/Stakeholders/StakeholderDetail/StakeholderDetail.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  TableRow,
+  TableCell,
+  Dialog,
+  DialogContent,
+  Divider,
+  IconButton,
+  Grid,
+} from '@material-ui/core';
+import PersonIcon from '@material-ui/icons/Person';
+import CloseIcon from '@material-ui/icons/Close';
+
+import StakeholderDialogHeader from './Header';
+import StakeholderGroups from './Groups';
+
+const useStyles = makeStyles({
+  flex: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  logo: {
+    width: 48,
+    height: 48,
+    paddingRight: 12,
+  },
+  pl: {
+    paddingLeft: 32,
+  },
+  closeButton: {
+    position: 'absolute',
+    right: 4,
+    top: 4,
+  },
+  my: {
+    margin: '16px 0',
+  },
+  noScroll: {
+    overflow: 'hidden',
+  },
+});
+
+export default function StakeholderDetail({ row, columns, child }) {
+  const {
+    type,
+    org_name,
+    first_name,
+    last_name,
+    image_url,
+    email,
+    phone,
+    website,
+    logo_url,
+    map,
+  } = row;
+  const rowData = {
+    ...row,
+    type: type || '',
+    org_name: org_name || '',
+    first_name: first_name || '',
+    last_name: last_name || '',
+    image_url: image_url || '',
+    email: email || '',
+    phone: phone || '',
+    website: website || '',
+    logo_url: logo_url || '',
+    map: map || '',
+  };
+
+  const [open, setOpen] = useState(false);
+  const classes = useStyles();
+
+  const openModal = () => {
+    setOpen(true);
+  };
+
+  const closeModal = () => {
```

Comment:
If `isEditing` is true, show a warning before closing to prevent users losing unsaved changes.